src/services/item/plugins/importExport/utils.ts

```diff
@@ -50,3 +51,55 @@ export const buildTextContent = (url: string, type: UnionOfConst<typeof ItemType
   }
   return `[InternetShortcut]\n${URL_PREFIX}${url}\n${APP_URL_PREFIX}1\n`;
 };
+
+const extractFileName = (itemName: string, extension: string) => {
+  const fullExtension = `.${extension}`;
+  const fileName = `${path.basename(itemName, fullExtension)}`;
+  return `${fileName}${fullExtension}`;
+};
+
+const extractExtension = ({ name, mimetype }: { name: string; mimetype?: string }): string => {
+  // slice to remove . character
+  const ext = path.extname(name).slice(1);
+  if (!ext && mimetype) {
+    return mime.extension(mimetype) || '';
+  }
+  return ext;
+};
+
+// use partial of item to be usable in backend
+export const getFilenameFromItem = (item: {
+  name: string;
+  type: ItemTypeUnion;
+  mimetype?: string;
+}): string => {
+  switch (item.type) {
+    case ItemType.APP: {
+      return extractFileName(item.name, 'app');
+    }
+    case ItemType.DOCUMENT: {
+      if (item.mimetype === 'text/html') {
+        return extractFileName(item.name, 'html');
+      }
```

Comment:
So it is not based on the `isRaw` setting ? The other test files makes it look it is.